app/controllers/user/sessions_controller.rb

```diff
@@ -9,9 +9,19 @@ class User::SessionsController < Devise::SessionsController
   # end
 
   # POST /resource/sign_in
-  # def create
-  #   super
-  # end
+  def create
```

Comment:
made these changes to create seperate path for admin and customers after login. 
otherwise it calls super in which at last ```after_sign_in_path_for``` is called therefore making same paths for all users

Reply:
done